### PR TITLE
User label

### PR DESCRIPTION
**Pull request by agagancarczyk**
Added endpoint for updating  a credential label for a user  
-> PUT /{realm}/users/{id}/credentials/{credentialId}/userLabel 
-> Spec https://www.keycloak.org/docs-api/15.0/rest-api/index.html

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@keycloak/keycloak-admin-client",
-  "version": "16.0.0-dev.55",
+  "version": "16.0.0-dev.56",
```
Comment (edewit) [inline package.json:3]: don't need to do this, this is done by the github action

**Review by edewit — CHANGES_REQUESTED** [review]
apart from the version update this looks good

File: package.json (hunk 1 above)
Comment (agagancarczyk) [reply package.json:3]: thanks @edewit!

**Review by jonkoops — APPROVED** [review]

**Comment by jonkoops** [comment]
@agagancarczyk Looks like the tests are failing

File: test/users.spec.ts [hunk 2]
```diff
@@ -199,6 +199,35 @@ describe('Users', function () {
...
+    const result = await kcAdminClient.users.getCredentials({
+      id: userId!,
+    });
+
+    expect(result.map(c => c.type)).to.include('password');
+
+    const credential = result[0];
+
+    kcAdminClient.setConfig({
+      requestConfig: {headers: {'Content-Type': 'text/plain'}},
```
Comment (jonkoops) [inline test/users.spec.ts:216]: Why would we have to set this header here? The request method itself should be responsible for setting the correct headers.

Comment (agagancarczyk) [reply test/users.spec.ts:216]: I'm trying to set it just for this special test case. I think we are globally setting the content-type to application/json but I'm trying to override it. Would you know a better way to do this @jonkoops ? Thanks!

Comment (jonkoops) [reply test/users.spec.ts]: Pushed a commit that should resolve this issue

**Review by agagancarczyk — COMMENTED** [review]
looks good, let's see

**Review by edewit — APPROVED** [review]
:+1: that is great

**Pull request merged**